apis/apis.json

```diff
@@ -35,11 +35,11 @@
     "id": "Google.Cloud.BigQuery.V2",
     "productName": "Google BigQuery",
     "productUrl": "https://cloud.google.com/bigquery/",
-    "version": "1.1.0",
```

Comment:
Is this not going to beta for a bit, before GA?

Reply:
No, I think we're happy with the change here (regionalization is the only change) so we're just going straight to GA.